lib/game.rb

```diff
@@ -0,0 +1,57 @@
+class Game
+  attr_reader :player1, :player2
+  def initialize(player1, player2)
+    @player1 = player1
+    @player2 = player2
+  end
+  
+  # Output game setup and prompt user to start game by entering 'GO'. 
+  
+  def start
+    puts "Welcome to War! (or Peace) This game will be played with 52 cards.\n"
+    puts "The players today are #{player1.name} and #{player2.name}.\n"
+    puts "Type 'GO' to start the game!"
+    user_response = gets.chomp
+    while user_response != "GO"
+      puts "Sorry, that was not a valid response.\n"
+      puts "Type 'GO' to start the game!"
+      user_response = gets.chomp
+    end
+    play
+  end
+  
+  # Set turn count to 0 and play out a game until the earlier of either player losing, or 1,000,000 turns
+  
+  def play
+    turn_count = 0
+    
+    until player1.has_lost? || player2.has_lost? do
+      turn_count += 1
+      turn = Turn.new(player1, player2)
```

Comment:
I really really like that for each turn you are creating a new `Turn` object. This is exactly how the game is supposed to flow programmatically.